runners/console/index.ts

```diff
@@ -5,7 +5,7 @@ import { Assertions } from "../../assertions";
 import { Playbook } from "../../engine/playbook";
 import { ConsolePlatform, AsyncProcess } from "./consoleInterfaces";
 import * as path from 'path';
-import * as fs from "fs";
+import * as fs from "fs-extra";
```

Comment:
Are the methods from the fs module (e.g. writeFileSync or appendFileSync) also available in fs-extra module?  I thought you have to add the fs-extra module additionally.

Reply:
https://www.npmjs.com/package/fs-extra
here it says, that from 'fs' is not necessary anymore